src/control/lib/netdetect/netdetect.go

```diff
@@ -149,10 +151,11 @@ type DeviceAffinity struct {
 
 // FabricScan data encapsulates the results of the fabric scanning
 type FabricScan struct {
-	Provider   string
-	DeviceName string
-	NUMANode   uint
-	Priority   int
+	Provider    string
+	DeviceName  string
+	NUMANode    uint
+	Priority    int
+	NetDevClass int32
```

Comment:
Should probably be uint32 considering all the values below are positive.

Reply:
Done.